_non-cosmos/polygon/assetlist.json

```diff
@@ -277,6 +277,46 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdt.svg"
         }
       ]
+    },
+    {
+      "description": "The PAGE token is used for actions in the PageDAO NFT literary ecosystem and for DAO governance.",
+      "type_asset": "erc20",
+      "address": "0x9ceE70895726B0ea14E6019C961dAf32222a7C2f",
+      "denom_units": [
+        {
+          "denom": "0x9ceE70895726B0ea14E6019C961dAf32222a7C2f",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "PAGE",
+          "exponent": 8
+        }
+      ],
+      "base": "0x9ceE70895726B0ea14E6019C961dAf32222a7C2f",
+      "name": "Page",
+      "display": "Page",
```

Comment:
You have a similar issue with the display denom not having been defined (I commented this under its addition in the ethereum assetlist.)